keysign/discover.py

```diff
@@ -42,13 +42,17 @@ def start(self):
         if self.bt_code and not self.stopped:
             # We try Bluetooth, if we have it
             log.info("Trying to connect to %s with Bluetooth", self.bt_code)
-            self.bt = BluetoothReceive(self.bt_port)
-            msg_tuple = yield self.bt.find_key(self.bt_code, self.mac)
-            key_data, success, message = msg_tuple
-            if key_data:
-                # If we found the key
-                returnValue((key_data, success, message))
-
+            # We try to see if Bluetooth was imported,
+            # else we log an event of missing Pybluez.
+            try:
+                self.bt = BluetoothReceive(self.bt_port)
+                msg_tuple = yield self.bt.find_key(self.bt_code, self.mac)
+                key_data, success, message = msg_tuple
+                if key_data:
+                    # If we found the key
+                    returnValue((key_data, success, message))
+            except:
```

Comment:
don't do such a very broad catch block. What error do you get when pybluez is not installed?  Catch that.  And maybe even put it in the log via `log.exception`.